lecture2gether-vue/src/components/Player.vue

```diff
@@ -1,15 +1,17 @@
 <template>
-    <video-player class="video-player-box"
-                  ref="videoPlayer"
-                  :options="playerOptions"
-                  :playsinline="true"
-                  :events="['seeked', 'ratechange']"
-
-                  @play="onPlayerPlay"
-                  @pause="onPlayerPause"
-                  @seeked="onPlayerSeeked"
-                  @ratechange="onPlayerRate">
-    </video-player>
+    <div>
+        <video-player class="video-player-box"
+                    ref="videoPlayer"
+                    :options="playerOptions"
+                    :playsinline="true"
+                    :events="['seeked', 'ratechange']"
+
+                    @play="onPlayerPlay"
+                    @pause="onPlayerPause"
+                    @seeked="onPlayerSeeked"
+                    @ratechange="onPlayerRate">
+        </video-player>
+    </div>
```

Comment:
Wrapping video-player in a div seems unnecessary to me. Is there a specific reason to do that?

Reply:
No the reason has been removed in a previous commit.